@here/harp-lines/lib/Lines.ts

```diff
@@ -48,9 +48,10 @@ const LINE_VERTEX_ATTRIBUTES: VertexDescriptor = {
         { name: "extrusionCoord", itemSize: 2, offset: 0 },
         { name: "position", itemSize: 3, offset: 2 },
         { name: "tangent", itemSize: 3, offset: 5 },
-        { name: "bitangent", itemSize: 4, offset: 8 }
+        { name: "bitangent", itemSize: 4, offset: 8 },
+        { name: "lineLength", itemSize: 1, offset: 12 }
```

Comment:
Can we not squeeze it somewhere else, e.g. position

Reply:
Please clarify, you mean to put it after position attribute?

Reply:
no, add it to the position attribute (i.e. the .w component of position) or alternatively into the tangent attribute. Having only one float in a vertex attribute is quite wasteful (see e.g.: https://stackoverflow.com/questions/21676280/glsl-packing-4-float-attributes-into-vec4)

Please also add documentation to the vertex attributes. The documentation is quite poor right now.

Reply:
Done.